models/groups.go

```diff
@@ -145,8 +145,8 @@ func (g *Group) MapCognitoDetails(groupDetails *cognitoidentityprovider.GroupTyp
 	g.Created = *groupDetails.CreationDate
 }
 
-// BuildSuccessfulJsonResponse builds the Group response json for client responses
-func (g *Group) BuildSuccessfulJsonResponse(ctx context.Context) ([]byte, error) {
+// BuildSuccessfulJSONResponse builds the Group response json for client responses
+func (g *Group) BuildSuccessfulJSONResponse(ctx context.Context) ([]byte, error) {
```

Comment:
This introduces a breaking change as the exported function name has changed. Check where this module is used elsewhere to confirm or reconsider.